OpenOversight/app/models/database_cache.py

```diff
@@ -10,9 +10,12 @@
 DB_CACHE = TTLCache(maxsize=1024, ttl=24 * HOUR)
 
 
-def get_model_cache_key(model: Model, update_type: str):
+def get_model_cache_key(model: Model, update_type: str) -> Any:
     """Create unique db.Model key."""
-    return hashkey(model.id, update_type, model.__class__.__name__)
+    if model is not None:
+        return hashkey(model.id, update_type, model.__class__.__name__)
+    else:
+        return hashkey(model.__class__.__name__, update_type, model.__class__.__name__)
```

Comment:
Since we know in the else branch that `model` is `None`, we could just use `hashkey(update_type)` or `hashkey("None", update_type)` to make that clearer.

Reply:
Thinking about it, perhaps it would be better if the first parameter could be either an instance or the class type. Then you would use the instance for everything that is cached per-instance (e.g. per department) and the class type for stuff that is cached across all instances of the same class (e.g. the departments per state)